src/getProfile.js

```diff
@@ -0,0 +1,40 @@
+import { fetchProfiles } from './utils';
+
+const HOUR = 60 * 60
+
+export const getProfile = async (request, env, ctx) => {
+  const body = await request.json();
+  const relayUrls = body.relays.map(url => (new URL(url)).origin);
+  const pubkey = body.pubkey;
+
+  if (!relayUrls || !pubkey) {
+    return new Response('INVALID_REQUEST', { status: 422 });
+  }
+
+  const setCache = (key, data) => env.PROFILES.put(key, data, {
```

Comment:
Could you link me to the docs for this type of cache object? why not use the cache API? https://developers.cloudflare.com/workers/runtime-apis/cache/

Reply:
We are using KV pairs: https://developers.cloudflare.com/workers/runtime-apis/kv/